server/app/services/export/JsonExporter.java

```diff
@@ -71,21 +70,28 @@ public Pair<String, PaginationResult<Application>> export(
     return export(programDefinition, paginationResult);
   }
 
-  public Pair<String, PaginationResult<Application>> export(
+  public String export(
```

Comment:
nonblocking: While you're in here, can we disambiguate between these two `export()` methods? Either with a javadoc and/or the name?

iiuc one is for the admin export view, and the other is for the API and provides pagination information.